dtao.c

```diff
@@ -135,8 +142,36 @@ parse_color(const char *str, pixman_color_t *clr)
 	return 0;
 }
 
+struct clickable {
+	uint32_t x1, y1, x2, y2;
+	uint8_t btn;
+	char *cmd;
+};
```

Comment:
Please move this after the enum ([L38](https://github.com/djpohly/dtao/blob/main/dtao.c#L38))